### PR TITLE
multi-region: conditional bootstrap

### PR DESCRIPTION
**Pull request by bernadinm**
This adds control to enable a bootstrap node on the remote region

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -55,10 +55,10 @@ resource "aws_key_pair" "deployer" {
 
 // Create a VPC and subnets
 module "dcos-vpc" {
-  source  = "dcos-terraform/vpc/aws"
-  version = "~> 0.1"
+  source  = "github.com/dcos-terraform/terraform-aws-vpc?ref=multi-region"
```
Comment (fatz) [inline main.tf]: needs to be changed before merge

File: main.tf [hunk 2]
```diff
@@ -70,7 +70,8 @@ module "dcos-vpc" {
 
 // Firewall. Create policies for instances and load balancers
 module "dcos-security-groups" {
-  source = "dcos-terraform/security-groups/aws"
+  source = "github.com/dcos-terraform/terraform-aws-security-groups?ref=multi-region"
```
Comment (fatz) [inline main.tf]: needs to be changed before merge

File: main.tf [hunk 3]
```diff
@@ -97,27 +98,26 @@ module "dcos-iam" {
 }
 
 module "dcos-bootstrap-instance" {
-  source  = "dcos-terraform/bootstrap/aws"
-  version = "~> 0.1"
+  source = "github.com/dcos-terraform/terraform-aws-bootstrap?ref=conditional-bootstrap"
```
Comment (fatz) [inline main.tf]: needs to be changed before merge

File: .gitignore [hunk 4]
```diff
@@ -1,9 +1,76 @@
-#  Local .terraform directories
-**/.terraform/*
+# Inspired by https://dcmnt.me/2bidtTq
```
Comment (fatz) [inline .gitignore]: can we please keep the .gitignore clean. A lot of entries which are not needed for this project

**Review by fatz — CHANGES_REQUESTED** [review]
have to review other modules first. But keeping this as changes requested because of non registry source

**Comment by fatz** [comment]
@bernadinm can you take care of the conflicts?

**Comment by bernadinm** [comment]
complete, thanks @fatz !

**Comment by fatz** [comment]
terraform fmt failed. @bernadinm please have a look at jenkins

**Comment by bernadinm** [comment]
This is blocked by this PR here: https://github.com/dcos-terraform/terraform-aws-bootstrap/pull/6

**Comment by bernadinm** [comment]
conflicts are taken care of, merging

**Pull request merged**